### PR TITLE
Use SQL INSTR() for CONTAINS expressions

### PR DESCRIPTION
**Pull request by superdupershant**
Implement the Tableau CONTAINS expression as INSTR() instead of the
current use of RLIKE(*.*). The INSTR SQL expression is implemented more
efficiently than RLIKE().

### CODE REVIEW

**Review by bogdanghit — APPROVED** [review]
LGTM, tdvt tests pass successfully after the change.

**Review by gregrahn — APPROVED** [review]
LGTM, pending tests passing

File: tableau-databricks/dialect.tdd [hunk 1]
```diff
@@ -72,6 +72,12 @@ limitations under the License.
         <argument type='date' />
     </function>
 
+    <function group='string' name='CONTAINS' return-type='bool'>
+        <formula>(INSTR(%1,%2) &gt; 0)</formula>
+        <argument type='str' />
+        <argument type='str' />
+    </function>
+
```
[the remark this answers is not in the document]
Comment (superdupershant) [reply tableau-databricks/dialect.tdd]: Yes I was attempting to keep the XML sorted by the name field, but moved and updated.

File: tableau-databricks/dialect.tdd [hunk 2]
```diff
@@ -149,6 +149,13 @@ limitations under the License.
         <argument type='localstr' />
         <argument type='date' />
     </date-function>
+
+    <function group='string' name='CONTAINS' return-type='bool'>
+        <formula>(INSTR(%1,%2) &gt; 0)</formula>
```
Comment (abehm) [inline tableau-databricks/dialect.tdd:154]: Even faster would be to `a LIKE '%b%'`

**Comment by superdupershant** [comment]
That would require escaping embedded % and _ in the column values.

**Pull request merged**